ci/scripts/target-location.sh

```diff
@@ -8,4 +8,23 @@
 set -e
 REPO=$(dirname "$0")/../..
 
-$REPO/bazelisk.sh cquery $1 --output starlark --starlark:file=$REPO/rules/output.cquery 2>/dev/null
+verbose='false'
+print_usage() {
+  printf "Usage: $0 [-v] <bazel target label>"
+}
+
+while getopts 'v' flag; do
+  case "${flag}" in
+    v) verbose='true' ;;
+    *) print_usage
+       exit 1 ;;
+  esac
+done
+
+REDIR=''
+if [ $verbose == 'false' ];
+then
+  $REPO/bazelisk.sh cquery ${@:$OPTIND:1} --output starlark --starlark:file=$REPO/rules/output.cquery 2>/dev/null
+else
+  $REPO/bazelisk.sh cquery ${@:$OPTIND:1} --output starlark --starlark:file=$REPO/rules/output.cquery
+fi
```

Comment:
Minor bash nits:

- Isn't the usual approach to `shift` in the getopts handler? Then you can just use `"$@"` rather than messing around with OPTIND
- It's a bit of a pity to have to write the command twice. That said, I couldn't figure out a way to get the `2>/dev/null` bit from a variable. :-( I guess we could put the command itself in a variable, but that's a bit overly complicated. Any clever ideas? (If not, no problem: this is fine too!)

Reply:
Thank you for the suggestions and review